core/src/main/java/org/apache/accumulo/core/file/rfile/PrintInfo.java

```diff
@@ -138,6 +146,19 @@ public String description() {
     return "Prints rfile info";
   }
 
+  @SuppressWarnings("unchecked")
```

Comment:
If you assign the unchecked code to a variable, you can add this suppression on just that assignment, so it doesn't mask other (fixable, avoidable) problems in the method.